test/README.md

```diff
@@ -0,0 +1,20 @@
+
+## Notes on reproducing
+
+With the model weights downloaded from Google Drive and unpacked into the `vit_model` directory below:
+
+```
+python test.py -w ../vit_model/vit_finetuned_Bal_CE_lr5e-05_epochs10 -o out -m 2 -f ../../plankton_ml/tests/fixtures/test_images/
+```
+
+OSError: Error no file named pytorch_model.bin, model.safetensors, tf_model.h5, model.ckpt.index or flax_model.msgpack found in directory ../vit_model/vit_finetuned_Bal_CE_lr5e-05_epochs10.
```

Comment:
I've belatedly realised that [in the Best-Models-18](https://drive.google.com/drive/folders/1r_0pP0YarZ8yT1-jImknkr_IelAxhWKH) subdirectory there are copies of the ViT and beit models with the `model.safetensors` file (I'd naively unzipped the .pt file and pointed at that), so having another go